rskj-core/src/main/java/org/ethereum/core/TransactionSet.java

```diff
@@ -41,7 +42,7 @@ public TransactionSet(Map<Keccak256, Transaction> transactionsByHash, Map<RskAdd
         this.transactionsByAddress = transactionsByAddress;
     }
 
-    public void addTransaction(Transaction transaction) {
+    public void addTransaction(Transaction transaction, SenderResolverVisitor senderResolver) {

```

Comment:
Maybe pass this as a constructor parameter

Reply:
I haven't did it just b/c of laziness, I'll amend it :)